README.md

```diff
@@ -64,7 +64,12 @@ module.exports = function(config) {
       // only render the graphic after all tests have finished.
       // This is ideal for using this reporter in a continuous
       // integration environment.
-      renderOnRunCompleteOnly: true // default is false
+      renderOnRunCompleteOnly: true, // default is false
+
+      // limit the number of lines of error shown
+      // No error occurs if this limit is longer than 
+      // the number of lines reported.
+      maxLogLines: 5 // default is 9001
```

Comment:
This line would also need to be added to the `npm version` readme (for some reason that part is missing from the `package.json`.

Also the `9001` needs to be `42` -- or whatever the final PR code will settle on.